.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing Guide
+
+Hello! We, the maintainers, are very pleased that you are interested in contributing to Verso. However, before you submit your contribution, please take a moment to read the [Code of Conduct](CODE_OF_CONDUCT.md) and the appropriate section for the contribution you plan to make:
```

Comment:
Instead `Verso`, put `Fortress`.

Reply:
link is also broken, no code of conduct file. (that could be copied from verso repo I guess.)